addons/binding/org.openhab.binding.evohome/README.md

```diff
@@ -0,0 +1,119 @@
+# evohome Binding
+
+This binding integrates the Honeywell evohome system. It uses your Honeywell Total Connect Comfort account to access your locations and heating zones.
+
+## Supported Things
+
+The binding supports the following things:
+
+* evohome Account
+* evotouch control display
+* Heating zones
+
+### evohome Account
+
+This thing functions as the bridge between all the other things. It contains your credentials and connects to the Honeywell web API. 
+
+### evotouch
+
+This thing represents the central display controller. It is used to view and change the current system mode.
+
+### Heating zone
+
+The heating zone thing represents the evohome heating zone. It displays the current temperature, the temperature set point and the status of the set point. It also allows you to permanently override the current temperature set point as well as canceling any active overrides. 
+
+## Discovery
+
+After setting up the evohome account, the evotouch and heating zones available to your account will be discovered after a manual scan.
+
+## Binding Configuration
+
+The evohome account needs to be configured with your username and password. This can be done via PaperUI or things file.
+
+## Thing Configuration
+
+Thing configuration is optional, it is easier to use discovery which will automatically add all your zones and displays to the inbox, once the account Thing is online.
```

Comment:
In this section, you should simply describe the config parameters in a table (see e.g. [here](https://github.com/openhab/openhab2-addons/tree/master/addons/binding/org.openhab.binding.gardena#account-configuration)) as they apply to Paper UI and textual configuration likewise - no need for examples, which should come below in the "full example" section.